scripts/tasks/verify-treeshakable.js

```diff
@@ -46,9 +49,7 @@ const input = process.argv[2];
 check(input)
     .then((res) => {
         if (res.isTreeShakable === false) {
-            console.error(
-                `Failed to fully treeshake ${input}. Remaining code after treeshaking:\n\n${res.code}`
-            );
+            console.error(`${res.code}\n❗️ Failed to fully treeshake ${input}`);
```

Comment:
I find it more helpful to be able to see the error right away when the check fails, rather than having to scroll up past a big ol' code dump.